src/cozy/react/components/browserIcons.js

```diff
@@ -3,11 +3,22 @@ import BrowserFirefoxIcon from 'cozy-ui/transpiled/react/Icons/BrowserFirefox'
 import BrowserChromeIcon from 'cozy-ui/transpiled/react/Icons/BrowserChrome'
 import BrowserEdgeChromiumIcon from 'cozy-ui/transpiled/react/Icons/BrowserEdgeChromium'
 
+import StoreChromeIcon from 'cozy/react/assets/store_chrome.svg'
+import StoreSafariIcon from 'cozy/react/assets/store_safari.svg'
+import StoreFirefoxIcon from 'cozy/react/assets/store_firefox.svg'
+
 const browserIcons = {
   safari: BrowserSafariIcon,
   firefox: BrowserFirefoxIcon,
   chrome: BrowserChromeIcon,
   'edge-chromium': BrowserEdgeChromiumIcon
 }
 
+export const extensionStoresIcons = {
+  chrome: StoreChromeIcon,
+  safari: StoreSafariIcon,
+  firefox: StoreFirefoxIcon,
+  'edge-chromium': StoreChromeIcon
```

Comment:
Don't we manage `brave` for instance in other places?

Reply:
Only place I found a reference to Brave was in old `Install` button : https://github.com/cozy/cozy-pass-web/blob/93e2d903f190b6c5570c4d5b134dac53184adbb8/src/app/vault/groupings.component.ts#L36-L51

But here I just got 3 icons, and by default Brave answers `Chrome` with `detect-browser`.

I can add a specific case if needed (like for the `Install` button
https://github.com/cozy/cozy-pass-web/blob/93e2d903f190b6c5570c4d5b134dac53184adbb8/src/app/vault/groupings.component.html#L157-L191

Reply:
I just realized that Opera is not handled

Reply:
I don't know if we should support them, let's see with @poupotte

Reply:
I changed the behavior, now the button is not displayed if browser is not supported.
Also if browser is supported but no icon is provided it now displays the generic laptop icon.